projects/ontimize-web-ngx/src/lib/components/table/o-table.component.ts

```diff
@@ -2138,6 +2139,12 @@ export class OTableComponent extends AbstractOServiceComponent<OTableComponentSt
     if (this.oTableQuickFilterComponent) {
       this.oTableQuickFilterComponent.setValue(void 0);
     }
+
+    if (this.filterBuilder) {
+      this.filterBuilder.clearFilter();
+      this.filterBuilder.triggerReload();
```

Comment:
Test if `triggerReload()` has no conflicts with an hipothetic query of developer if it is registered to onFilterByColumnChange emmiter

Reply:
Its necessary emit onFilterByColumnChange event to update  the state of the column filter icon
![image](https://user-images.githubusercontent.com/32088627/179958495-54e551b2-5fa3-4fdf-b5b4-9e8ac8e4e0f0.png)